src/design_notes/autoref_ops.md

```diff
@@ -0,0 +1,29 @@
+# Autoref/Autoderef in operators
+
+Rust permits overriding most of the operators (e.g., `+`, `-`, `+=`). In part
+due to `Copy` types not auto-dereferencing, it is common to have `T + &T` or `&T
++ &T`, with potentially many levels of indirection on either side of the
```

Comment:
The "+" at the first character of the second line will mark that line as a row of list.
```suggestion
due to `Copy` types not auto-dereferencing, it is common to have `T + &T`
or `&T + &T`, with potentially many levels of indirection on either side of the
```